packages/block-editor/src/components/block-navigation/list.js

```diff
@@ -2,23 +2,28 @@
  * External dependencies
  */
 import { isNil, map, omitBy } from 'lodash';
-import classnames from 'classnames';
 
 /**
  * WordPress dependencies
  */
-import { Button, VisuallyHidden } from '@wordpress/components';
+import { Slot, Fill } from '@wordpress/components';
 import {
-	__experimentalGetBlockLabel as getBlockLabel,
-	getBlockType,
-} from '@wordpress/blocks';
-import { __ } from '@wordpress/i18n';
+	Children,
+	cloneElement,
+	useContext,
+	createContext,
+} from '@wordpress/element';
 
 /**
  * Internal dependencies
  */
-import BlockIcon from '../block-icon';
+import BlockNavigationListItem from './list-item';
 import ButtonBlockAppender from '../button-block-appender';
+import { BlockListBlockContext } from '../block-list/block';
+
+export const BlockNavigationContext = createContext( {
+	withBlockNavigationSlots: true,
```

Comment:
My feeling is that it might be better to default this to false, so that it only has to be specified as `true` in the nav menus page. Changing less code in the post editor would be a positive.

Reply:
Do you think we should only restrict that to nav menus page? This only disables those slots in the "global" navigation here:

<img width="752" alt="80591719-72544a00-8a1e-11ea-8c9b-152867159ab0" src="https://user-images.githubusercontent.com/205419/81058451-bf2c9a80-8ece-11ea-94a8-5d49b3c743a8.png">

But they remain active for example in the block inspector. The original issue actually was based of the post editor: https://github.com/WordPress/gutenberg/issues/20748

Reply:
I'm ok with either. The navigation block is experimental, so I think it's fine to keep it active there.

I think we'll end up removing those other navigators within the block eventually anyway, this seems like a great way forwards that would make them irrelevant - https://github.com/WordPress/gutenberg/issues/22113.